plugin/route53/setup.go

```diff
@@ -122,8 +130,13 @@ func setup(c *caddy.Controller) error {
 			}
 		}
 
-		client := f(opts)
 		ctx, cancel := context.WithCancel(context.Background())
+		cfg, err := config.LoadDefaultConfig(ctx, cfgOpts...)
```

Comment:
This can’t happen here, it fails in tests if the relevant AWS setup isn’t present.